### PR TITLE
fix(integrationRuntimes,logAnalyticsWorkspace,sqlServers): several fixes

### PR DESCRIPTION
**Pull request by hjaraujof**
## Changes/solution

- Removed several sensitive data fields from schemas
- Removed spread operations to prevent unmapped field errors
- Format fixes

### CODE REVIEW

File: src/services/integrationRuntimes/format.ts [hunk 1]
```diff
@@ -113,47 +110,11 @@ export interface RawAzureIntegrationRuntimeUnion {
...
-    store: {
-      type: secret?.store?.type,
-      referenceName: secret?.store?.referenceName,
-      parameters: formatParameters(secret?.store?.parameters),
-    },
-    secretName: Object.values(secret?.secretName ?? {}).join(''),
-    secretVersion: Object.values(secret?.secretVersion ?? {}).join(''),
-  }
-}
+// function useInterfaceToExtractData<ExpectedDataType>(
```
Comment (tyler-dunkel) [inline src/services/integrationRuntimes/format.ts]: is this still needed?

File: src/services/logAnalyticsWorkspace/format.ts [hunk 2]
```diff
@@ -18,9 +18,13 @@ export default ({
     resourceGroupId,
     provisioningState,
     customerId,
-    sku,
-    features,
-    workspaceCapping,
+    sku: { name, lastSkuUpdate },
```
Comment (tyler-dunkel) [inline src/services/logAnalyticsWorkspace/format.ts:21]: do these need default values?

File: src/services/integrationRuntimes/format.ts (hunk 1 above)
Comment (hjaraujof) [reply src/services/integrationRuntimes/format.ts]: No, I can remove it. I was looking to implement a routine that received the resulting type/interface and the object, allowing us to remove unwanted properties using the generated schema interface/type. But this needs some research to do it properly without introducing new bugs.

File: src/services/logAnalyticsWorkspace/format.ts (hunk 2 above)
Comment (hjaraujof) [reply src/services/logAnalyticsWorkspace/format.ts:21]: Not really. Although it is optional on the type, both properties are always populated.

**Review by tyler-dunkel — APPROVED** [review]

**Pull request merged**